deployment/helm/ditto/values.yaml

```diff
@@ -169,8 +169,15 @@ dbconfig:
 ingress:
   # enabled whether Ingress should be enabled as alternative to the contained nginx
   enabled: false
+  # namespace for ingress controller, must exist
+  namespace: ingress-nginx
+  # Ingress-NGINX version
+  nginxIngressVersion: v1.8.0
+  # Nginx Version
+  nginxVersion: "1.21.6"
+
   # className is the 'ingressClassName' to configure in the Ingress spec
-  className: nginx
+  className: ingress-nginx
```

Comment:
Is this an officially supported Ingress `className`? 
I did quickly search for it and only found that the nginx-ingress-controller uses `nginx` (as it currently also is configured).

Reply:
Currently `ingress.className` must match `ingress.controller.namespace` to ingress really working, which I guess is not the best approach. They should probably be independent parameters, so I'm trying to separate them.

Reply:
Fixed. Two independent parameters. Default className is nginx.

Reply:
Actually changing controller class name not working at the moment. Reverted back.

Reply:
Hm, different namespaces are now also working, looks like a glitch in k8s :confused: